### PR TITLE
Make VOD playback work again.

### PR DESCRIPTION
**Pull request by fredrikaubert**
Propably some changes made in nrk api causing VOD playback to not work for me on kodi 19. Only live streams worked. This pull-request should make browsing and starting streams from categories work again.

### CODE REVIEW

**Comment by petterreinholdtsen** [comment]
[fredrikaubert]
> Propably some changes made in nrk api causing VOD playback to not work
> for me on kodi 19. Only live streams worked. This pull-request should
> make browsing and starting streams from categories work again.

Nice.  Perhaps you want to maintain the kodi package?  @tamland have
stated he no longer want to.

-- 
Happy hacking
Petter Reinholdtsen

File: nrktv.py
Comment (CaptainTyphoo) [inline nrktv.py:22]:
```suggestion

```
please remove the unnecessary import

File: nrktv.py
Comment (CaptainTyphoo) [inline nrktv.py:143]:
```suggestion
        manifest = _get('/playback/manifest/{}'.format(r.get('id')))
```
please use the already existing function for GET requests

File: nrktv.py
Comment (CaptainTyphoo) [inline nrktv.py:144]:
```suggestion

```
please do not add commented out code

File: nrktv.py
Comment (CaptainTyphoo) [inline nrktv.py:148]:
```suggestion

```
please do not add commented out code

File: subs.py
Comment (CaptainTyphoo) [inline subs.py:42]: 1. Is there no conversion from vtt to srt required?

2. Please make the the function private:
```suggestion
def _vtt_to_srt(content):
```

File: subs.py
Comment (CaptainTyphoo) [inline subs.py:46]: Please remove the _ttml_to_srt function as it is no longer used.

File: subs.py
Comment (CaptainTyphoo) [inline subs.py:35]:
```suggestion
    content = _vtt_to_srt(subs)
```
please make the the function private

**Review by CaptainTyphoo — CHANGES_REQUESTED** [review]
Thank you very much for your contribution!
Could you please incorporate my review comments?
I will merge your changes to master after my comments have been resolved.

File: subs.py
Comment (fredrikaubert) [reply subs.py]: Don't really know if there are any conversion needed. I'm not really familiar with the different subtitle-formats. 

I've been running this code for some time watching nrk content, and I've never noticed any issues with the subs.

**Comment by fredrikaubert** [comment]
> Nice. Perhaps you want to maintain the kodi package? @tamland have stated he no longer want t

Thanks, but no thanks. :-) I don't have the pastime to commit to maintaining right now.

**Review by CaptainTyphoo — APPROVED** [review]
Looks good!

**Comment by andrea99boban** [comment]
I made this addon (after pull #71) to work under Kodi 20.0, few lines changed in 2 files. Is this of interest to evaluate being merged in?

**Comment by petterreinholdtsen** [comment]
[andrea99boban]
> I made this addon (after pull #71) to work under Kodi 20.0, few lines
> changed in 2 files. Is this of interest to evaluate being merged in?

I would love to see a new and working version in the Kodi add-on
repositories, at least.  But I am just a random user passing by.  I
suspect a fork is needed as the original developer @tamland have stated
he is busy elsewhere these days.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by andrea99boban** [comment]
fork done, also as pull #73 here

**Comment by solbu** [comment]
> fork done

As a service to others, could you link to your fork, making it easier to find? :-)

**Comment by solbu** [comment]
And if you plan on being the official maintainer, you should open the issues feature on your fork.

**Pull request merged**
